src/Forum/utils.ts

```diff
@@ -0,0 +1,70 @@
+import { env } from 'decentraland-commons'
+import { shorten } from '../utils/address'
+import { CollectionAttributes } from '../Collection'
+import { FullItem } from '../Item'
+import { ForumPost } from './Forum.types'
+
+// These methods are suspiciously similar to https://github.com/decentraland/builder/blob/master/src/modules/forum/utils.ts
+// It should be deleted from there and used here once we tackle https://github.com/decentraland/builder/issues/1754
+// Keep in mind the TODO above getItemEditorUrl
+
+const BUILDER_URL = env.get('BUILDER_URL', '')
+const BUILDER_SERVER_URL = env.get('BUILDER_SERVER_URL', '')
+
+export function buildCollectionForumPost(
+  collection: CollectionAttributes,
+  items: FullItem[]
+): ForumPost {
+  // We only post in English
+  return {
+    title: `Collection '${collection.name}' created by ${shorten(
+      collection.eth_address
```

Comment:
This slipped thru. I guess the third party name is a good option as we don't have the actual collection creator. It hurts a bit to have to add a conditional argument for TPs :/

Reply:
I would write `Third Party collection ${collection.name} with URN: ${urn}`
The address of the creator is there to avoid issues with collections that seemed to have the same name due to whitespaces being removed.